main.go

```diff
@@ -930,7 +930,10 @@ func buildBlacklist(files []*ast.File, info *types.Info, pkg *types.Package) map
 	blacklist := make(map[types.Object]struct{})
 
 	reflectBlacklist := func(node ast.Node) bool {
-		expr, _ := node.(ast.Expr)
+		expr, ok := node.(ast.Expr)
+		if !ok {
+			return true
+		}
```

Comment:
did you change this for any reason in particular? as far as I can tell the previous code was fine; if `expr == nil`, then `named == nil` and we `return true` anyway.

you can add an inline comment to make that more obvious though, like:

```
expr, _ := node.(ast.Expr) // note that info.TypeOf(nil) will just return nil
```

Reply:
I just changed it to make it more obvious yeah, I can change it back and add that comment

Reply:
Yeah I agree. I think one line is still better than four if we can still make it obvious, though :)